README.md

```diff
@@ -1 +1,28 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-World
+This Project is created purposely to learn and understand concept of version control.
+
+## Built With
+Computer programm
+
+## Requirements
+Github
+
+## Getting Started
+This project was setup using Visual Studio Code Editor. A linter was as well setup for the HTML and CSS files.
+Irrelevant files were specified in .gitignore file including the test file.
+
+## Prerequisite
+How to use a Computer
+
+## Setup
+
+## Install
+
+## Usage
+
+## Run tests
+
+## Deployment
+
+## Author
+* GitHub: @OyePriscilla
```

Comment:
- Well done adding a README, but you should consider using [MicroverseTemplate](https://github.com/microverseinc/readme-template) and don't forget to customize it as per the requirement.